homeworks/dmytro.shevchenko_dmytro-shevchenko1/GamesProject/style.css

```diff
@@ -0,0 +1,26 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap");
+
+main {
+  width: 1175px;
+}
+
+.intro {
+  color: #221F1F;
+  width: 575px;
+  font-size: 22px;
+  font-weight: 400;
+  font-family: Roboto, sans-serif;
+  line-height: 32px;
+}
+
+h1 {
+  color: #221F1F;
+  font-size: 48px;
+  font-weight: 700;
+  line-height: 32px;
+  margin-block-start: 0;
```

Comment:
Hey, @dmytro-shevchenko1, do you really need this prop https://developer.mozilla.org/en-US/docs/Web/CSS/margin-block-start?
Maybe just `margin-top`?

Reply:
Hey, @vladyslav-yermolin-moc, found this prop in DevTools. I'll change it. Next time, I'm going to be more careful when choosing properties. :)